packages/replay-next/src/hooks/useForceUpdate.ts

```diff
@@ -0,0 +1,9 @@
+import { useCallback, useState } from "react";
+
+export function useForceUpdate() {
```

Comment:
I didn't end up using this because I went with `useSyncExternalStore` instead. Seems like it's maybe worth adding though in case?